lib/main.js

```diff
@@ -115,6 +115,8 @@ const Hydrogen = {
           this.handleKernelCommand({ command: "interrupt-kernel" }),
         "hydrogen:restart-kernel": () =>
           this.handleKernelCommand({ command: "restart-kernel" }),
+        "hydrogen:refresh-all-result-bubbles": () =>
```

Comment:
This command redraw(refresh) only existing bubbles, so `bubble` should be included as part of command name.
`restart-kernel-and-re-evaluate-bubbles`? but its's verbose.

`restart` is just a underlying detail to achieve refresh, if kernel provide more light way to cleanup env( like node's vm.createContext ), it also can be used to achieve refresh.
So, in my opinion I prefer current name. how do you think?